cats/cats-sql/src/test/groovy/com/netflix/spinnaker/cats/sql/SqlCacheSpec.groovy

```diff
@@ -96,6 +97,23 @@ class SqlCacheSpec extends WriteableCacheSpec {
     null                                                   || null             || "1=1"
   }
 
+  @Unroll
+  def 'max length of table name is checked'() {
+    when:
+    def realName = ((SqlCache) cache).checkTableName("cats_v1_", name, suffix)
+
+    then:
+    realName == expected
+
+    where:
+    name              || expected                                                             || suffix
```

Comment:
Style nit: Spock tests usually split inputs with a singe pipe `|`from the outputs with a double pipe `||`. http://spockframework.org/spock/docs/1.0/data_driven_testing.html#_syntactic_variations